src/operator/OrgOverlay.tsx

```diff
@@ -111,8 +120,26 @@ const OrgOverlay: FC = () => {
                       text="View Usage Dashboard"
                       target={LinkTarget.Blank}
                       className="overlay-button--link"
+                      style={{marginRight: '12px'}}
                       href={`${TOOLS_URL}orgs/5d59ccc5163fc318/dashboards/0988da0fd78a7003?vars%5Borgid%5D=${orgID}`}
```

Comment:
Consider moving the styles into an object outside the component to avoid a re-render.